### PR TITLE
#159072827 Setup eslint

### PR DESCRIPTION
**Pull request by veeqtor**
#### What does this PR do?
Setup linting with eslint 

#### Description of Task to be completed?

- install eslint
- configure eslint to extend `Airbnb`
- fix all eslint warnings

#### How should this be manually tested?
- After cloning the repo, `cd` into the directory
- Run `npm install` and `npm run start:dev`
- Using Postman, test the defined endpoints `/` and `/api/v1`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159072827](https://www.pivotaltracker.com/story/show/159072827)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "airbnb",
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true,
+        "mocha": true
+    }
+}
```
Comment (madeofhuman) [inline .eslintrc]: Add new line, else there'll be red, squiggly lines, and nobody wants red, squiggly lines.

File: package.json [hunk 2]
```diff
@@ -29,4 +33,4 @@
     "url": "https://github.com/chukwuemekachm/Book-Rest-Api/issues"
   },
   "homepage": "https://github.com/chukwuemekachm/Book-Rest-Api#readme"
-}
+}
```
Comment (madeofhuman) [inline package.json]: Add new line, else there'll be red, squiggly lines, and nobody wants red, squiggly lines.

File: .eslintrc (hunk 1 above)
Comment (chukwuemekachm) [reply .eslintrc]: Please, what does the "env" property do?
Setting these properties to true turns off certain rules.
Can you just extend "Airbnb"?

File: package.json (hunk 2 above)
Comment (veeqtor) [reply package.json]: Kindly give your feedback in ASK format.. Thank you.😄

**Pull request merged**